apps/meteor/server/services/voip/connector/asterisk/ami/CommandFactory.ts

```diff
@@ -7,33 +8,32 @@
  * they will be executed by |PJSIPEndpoint| class.
  */
 import { Db } from 'mongodb';
+import { Commands } from '@rocket.chat/core-typings';
 
-import { Command } from '../Command';
-import { Commands } from '../Commands';
 import { ACDQueue } from './ACDQueue';
 import { PJSIPEndpoint } from './PJSIPEndpoint';
 import { Logger } from '../../../../../lib/logger/Logger';
 import { ContinuousMonitor } from './ContinuousMonitor';
+import { PingCommand } from './Ping';
 
 export class CommandFactory {
 	static logger: Logger = new Logger('CommandFactory');
 
-	static getCommandObject(command: Commands, db: Db): Command {
+	static getCommandObject(command: keyof typeof Commands, db: Db): PJSIPEndpoint | ACDQueue | ContinuousMonitor | PingCommand {
 		this.logger.debug({ msg: `Creating command object for ${Commands[command]}` });
```

Comment:
```suggestion
		this.logger.debug({ msg: `Creating command object for ${Commands[command].toString()}` });
```

Reply:
Earlier it was printing enum position (i.e. `1`, `2` ) rather than the command name